docs/current/742989-cookbook.md

```diff
@@ -625,6 +625,44 @@ Set the Hashicorp Vault URI, namespace, role and secret identifiers as host envi
 
 [Learn more](./guides/723462-use-secrets.md)
 
+
+## Error handling
+
+### Aborting with unhandled error
+
+The following code listing demonstrates how to abort the script with an unhandled error, which allows debugging with the stack trace of the crash during development.
+
+```go file=./cookbook/snippets/error-handling/unhandled/main.go
+```
+
+### Aborting gracefully
+
+The following code listing demonstrates how to abort the script without crashing.
+
+```go file=./cookbook/snippets/error-handling/aborting/main.go
+```
+
+### Exit code and unexpected errors
```

Comment:
```suggestion
### Handle exit codes and unexpected errors
```

Reply:
"exit codes" (plural) reads funny to me. The use case is to handle an exit code (singular), it just so happens that the snippet demonstrates multiple ways you can do that.

Reply:
Modified